pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1007,7 +1007,7 @@ def test_cumsum(self, data, expected, numpy):
                 np.cumsum(SparseArray(data), out=out)
         else:
             axis = 1  # SparseArray currently 1-D, so only axis = 0 is valid.
-            msg = "axis\\(={axis}\\) out of bounds".format(axis=axis)
+            msg = f"axis\\(={axis}\\) out of bounds"
```

Comment:
do "f" and "r" prefixes play nicely together?

Reply:
I will need to add repr won't I since fstring defaults to str()?

Reply:
@simonjayhawkins ?

Reply:
I would prefer to use re.escape than use raw string

Reply:
Like so?

`re.escape(f"axis(={axis}) out of bounds")`

Reply:
that looks correct. does it work?

Reply:
@simonjayhawkins

Reply:
It passes the git diff test and compiles, I am unsure how to test that exact line unfortunately. Trying to learn but that is beyond me. This is the biggest project ive worked on

Reply:
This is part of a test. so OK if tests passing.